Create Language-Version-History.md

```diff
@@ -0,0 +1,104 @@
+Features Added in F# Language Versions
+====================
+
+# F# 1.0
+
+- Discriminated unions
+- Records
+- Tuples
+- Pattern matching
+- Type abbreviations
+- Object programming
+- Structs
+- Signature files
+- Scripting files
+- Imperative programming
+- Modules (no functors)
+- Nested modules
+- .NET Interoperability
+
```

Comment:
F# 1.1: https://docs.microsoft.com/en-us/archive/blogs/dsyme/a-taste-of-whats-new-in-f-1-1